### DISCUSSION TITLE
Tailwind CSS v1.6 is out!

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#2088] [Show and tell]
Hey! We just released Tailwind CSS v1.6 with a handful of exciting new features:

- [Animation support](https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.6.0#animation-support)
- [New prefers-reduced-motion variants](https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.6.0#new-prefers-reduced-motion-variants)
- [New overscroll-behavior utilities](https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.6.0#new-overscroll-behavior-utilities)
- [Generate your CSS without an input file](https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.6.0#generate-your-css-without-an-input-file)

For all the details, [check out the complete release notes on GitHub](https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.6.0), and let us know what you think!

**Comment by royvanv** [comment]
Didn't even need to create an issue to update the version number on the website 😂

**Comment by deadcoder0904** [comment]


**Reply by royvanv** [reply]
That is correct, not a typo.

**Reply by deadcoder0904** [reply]
That was so dumb of me 🤦‍♂️

**Comment by oomusou** [comment]
I use `npx tailwindcss build -o compiled.css` in Vue CLI project.

in `src/main.js`

```
import Vue from 'vue'
import App from './App.vue'
import './compiled.css'

Vue.config.productionTip = false

new Vue({
  render: h => h(App),
}).$mount('#app')
```

when  I build for production, I can't see PuregeCSS enable to purge `compiled.css`.

How can we use `compiled.css` with PuregeCSS ?
Thanks a lot

**Comment by spaceemotion** [comment]
Regarding "Generate your CSS without an input file"; how would I do this when I want to use custom PostCSS plugins? Right now I have to call `postcss app.css > dist/app.css` by hand, without calling tailwind-cli directly. Since PostCSS requires an input file, this does not seem to be helpful then?

(I use plugins like cssnano for production output).

**Reply by adamwathan** [reply]
Only works with our CLI tool. If using PostCSS directly then you still need an input file, or you can just point it at the input file we ship in `./node_modules/tailwindcss/tailwind.css`.

**Comment by bennettdams** [comment]
No one [created a PR to add `overflow-behavior` to the docs](https://github.com/tailwindlabs/tailwindcss.com/pulls), how to keep track for that now that it is released?

**Reply by adamwathan** [reply]
I'm planning to add the docs myself in the near future.

**Comment by yangjinjun3** [comment]
Hey，how to support ": after "pseudo-class  and  !important？
